test/conformance/runtime/cmd_args_test.go

```diff
@@ -40,7 +40,7 @@ func TestCmdArgs(t *testing.T) {
 	t.Parallel()
 	clients := test.Setup(t)
 
-	cmds := []string{"/ko-app/runtime", "abra"}
+	cmds := []string{"/usr/bin/runtime", "abra"}
```

Comment:
This code is not downstream's but upstream's so we need to patch to the code when running the CI on `release-next` branch.
Could you please send the patch against `release-next` branch for test and add the patch in `main` branch before changing the `release-x` branch?

Reply:
👍 We do not need to change the path. I just wanted to test if it would work, to be in line with eventing. Will change it back to `/ko-app`.

I did the changes to main locally, but wanted a build with actual images to test. But I'll do it against `release-next`.

Reply:
> I did the changes to main locally, but wanted a build with actual images to test. But I'll do it against release-next.

Ah, I see! That makes sense to send the PR against `release-x` before `main`.